package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "1.7.19-create-booking",
+  "version": "1.7.20-ctsc-role",
```

Comment:
No need to increase the version number, especially if we are just adding a line of code, should be 1.7.19-ctsc-role